test_tululbot.py

```diff
@@ -215,6 +215,31 @@ def test_leli_command_with_multiword_term(app):
         )
 
 
+def test_quote(app):
+    payload = {
+        'update_id': 12345,
+        'message': {
+            'message_id': 100,
+            'text': '/quote',
+            'chat': {
+                'id': 1
+            }
+        }
+    }
+
+    rv = app.post('/', data=json.dumps(payload),
+                  content_type='application/json')
+
+    assert rv.status_code == 200
+    json_response = json.loads(rv.get_data(as_text=True))
+    assert json_response['method'] == 'sendMessage'
+    assert json_response['chat_id'] == 1
+    # Fix the test later, baby
+    # assert json_response['text'] == expected_text
```

Comment:
Just realized that you must mock `.refresh_cache()` (not `.retrieve_random()`) for this test. Otherwise, this test will actually trigger a GET request to `cdn.rawgit.com`.

What do you think @fushar ?